pkg/jwt/middleware.go

```diff
@@ -0,0 +1,23 @@
+package jwt
+
+import (
+	"errors"
+)
+
+type JWTParserMiddleware = func() error
```

Comment:
For type definitions, it's more appropriate to remove the = sign. In this case, this is actually an alias and not a type definition.

Reply:
+1